clmm/modbackend/numcosmo.py

```diff
@@ -0,0 +1,186 @@
+# NumCosmo implementation of CLMModeling
+
+import gi
+gi.require_version('NumCosmo', '1.0')
+gi.require_version('NumCosmoMath', '1.0')
+from gi.repository import NumCosmo as Nc
+from gi.repository import NumCosmoMath as Ncm
+
+from .cluster_toolkit_patches import _patch_comoving_coord_cluster_toolkit_rho_m
+
+import math
+import numpy as np
+import warnings
+
+from . import func_layer
+from . func_layer import *
```

Comment:
please do not import `*`. Either import a module and use it or make specific imports

Reply:
See above.